deploy/operator/kustomization.yaml

```diff
@@ -8,5 +8,6 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: monitoring-stack-operator
+  newName: quay.io/rhobs/monitoring-stack-operator
```

Comment:
@sthaha we can update here so that deployment works? i was trying to deploy existing image

Reply:
you want to use OLM  for deployment instead of `kubectl create -f` . e.g 

```
 make operator-image bundle-image operator-push bundle-push IMAGE_BASE=local-registry:30000/monitoring-stack-operator VERSION=0.0.0-ci

./tmp/bin/operator-sdk run bundle local-registry:30000/monitoring-stack-operator-bundle:0.0.0-ci --install-mode AllNamespaces --skip-tls --namespace operators --verbose | tee tmp/olm.log
```
note: `local-registry` can also be used for pushing operator-images

Reply:
I was not using olm. I just ran kubectl create since image was not in docker hub it was erroring out when I updated the image to quay.io it was working. Was the manifest meant to use for local deployment?

Reply:
You will need to install OLM anyway for the operator to work properly since it creates a grafana subscription and requires OLM. 

For deployment of the operator into the cluster, my recommendation will be to create the bundle which packages all required CRD,  roles and deployments and use `operator-sdk run bundle` to run the operator. 

See: how e2e-tests workflow which installs the operator and then runs test works.

Reply:
If this is all too confusing I think we could create a make target `operator-deploy` which does this. Also update the docs on how to run deploy the operator in a cluster.

Reply:
Thanks Sunil. I will update the docs once I try this out